lib/files/file_manager.ts

```diff
@@ -122,7 +122,9 @@ export class FileManager {
         }
         // if the suffix does not match the executable,
         // the binary is something like: .exe and .zip
-        else if (existFile.indexOf(binary.suffix(osType, arch)) === -1) {
+        else if (
```

Comment:
This is an extra check... current implementation of this needs some fixing. This is because downloading a file before 2.23 suffix is mac32.zip and this method is returning mac64.zip. Unfortunately, when we get the suffix the version is not set.

Not having extra checks:
```
webdriver-manager status

chromedriver versions available: 2.20 [last], 2.20mac32.zip, 2.26 [default]
```

Adding a TODO(cnishina) to fix this.